src/stack_pr/cli.py

```diff
@@ -948,6 +948,10 @@ def land_pr(e: StackEntry, remote: str, target: str):
         input=pr_body.encode(),
     )
 
+def abandon_pr(e: StackEntry):
```

Comment:
One more comment: the code is organized in sections corresponding to top-level command (LAND/VIEW/SUBMIT/ABANDON). So this function probably should be moved to the "ABANDON" section (after line 1040).